pkg/mgrconfig/config.go

```diff
@@ -168,6 +168,11 @@ type Config struct {
 	// Regexps are matched against bug title, guilty file and maintainer emails.
 	Interests []string `json:"interests,omitempty"`
 
+	// Path to the strace binary compiled for the target architecture.
+	// If set, for each reproducer syzkaller will run it once more under strace and save
+	// the output.
+	StraceBin string `json:"strace_bin"`
```

Comment:
Do distributions provide cross-arch strace binaries? How are they named? I can't find any.
Or we will need to compile strace for each arch ourselves?

Reply:
We need static strace binary and I haven't managed to find any public repo with them. So we need to compile strace ourselves anyway.

Re. cross-arch -- I didn't find it either. There are so called [multiple personalities](https://man7.org/linux/man-pages/man1/strace.1.html#MULTIPLE_PERSONALITIES_SUPPORT), but they're pretty limited. And I didn't manage to compile an strace binary with them, actually.